.github/workflows/BlackFormat.yml

```
@@ -1,4 +1,4 @@
-name: Format Validation (Black)
+name: Fusion - Format Validation
```

```suggestion
name: Fusion - Black Format Validation
```

Tiny nitpick, should be consistent with other formatting workflow.

https://github.com/Autodesk/synthesis/blob/00bb1addcffae3eb14e7c00ae895d7cf48943232/.github/workflows/FissionESLintFormat.yml#L1

---

Right, I thought we didn't called it ES Lint.